### PR TITLE
referral email improvements

### PR DESCRIPTION
**Pull request by camhashemi**
@ryanpbrewster thanks!

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/payments/CreditRewardCommander.scala [hunk 1]
```diff
@@ -204,10 +204,10 @@ class CreditRewardCommanderImpl @Inject() (
     val referredOrg = orgRepo.get(referredOrgId)
     val referrerOrg = orgRepo.get(referrerOrgId)
     val adminEmails = orgMembershipRepo.getByRole(referrerOrgId, OrganizationRole.ADMIN).flatMap(adminId => Try(emailAddressRepo.getByUser(adminId)).toOption)
-    val subject = s"You earned a $$100 credit for ${referrerOrg.name} on Kifi"
+    val subject = s"You earned a ${orgReferrerCredit.toDollarString} credit for ${referrerOrg.name} on Kifi"
     val htmlBody =
       s"""
-         |Your team, <a href="https://www.kifi.com/${referrerOrg.handle.value}">${referrerOrg.name}</a>, earned a $$100 credit from ${referredOrg.name}.
+         |Your team, <a href="https://www.kifi.com/${referrerOrg.handle.value}">${referrerOrg.name}</a>, earned a ${orgReferrerCredit.toDollarString} credit from <a href="https://www.kifi.com/${referredOrg.handle.value}">${referredOrg.name}</a>.
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/payments/CreditRewardCommander.scala:210]: If you want be even cooler you can use `<a href="${pathCommander.pathForOrg(referrerOrg).absolute}">`

Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/payments/CreditRewardCommander.scala:210]: Not sure how necessary that is, though.

**Comment by ryanpbrewster** [comment]
:+1: sweet!

File: kifi-backend/modules/shoebox/app/com/keepit/payments/CreditRewardCommander.scala (hunk 1 above)
Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/payments/CreditRewardCommander.scala:210]: I'm not sure I'm THAT cool

**Pull request merged**